tests/system/Router/RouterTest.php

```diff
@@ -212,20 +212,28 @@ public function testAutoRouteFindsControllerWithFileAndMethod(): void
         $this->collection->setAutoRoute(true);
         $router = new Router($this->collection, $this->request);
 
-        $router->autoRoute('myController/someMethod');
+        copy(TESTPATH . 'system/Router/Controllers/MyController.php', APPPATH . 'Controllers/Mycontroller.php');
```

Comment:
```suggestion
        copy(TESTPATH . 'system/Router/Controllers/Mycontroller.php', APPPATH . 'Controllers/Mycontroller.php');
```

```
4) CodeIgniter\Router\RouterTest::testAutoRouteFindsControllerWithFileAndMethod
ErrorException: copy(/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Router/Controllers/MyController.php): Failed to open stream: No such file or directory

/home/runner/work/CodeIgniter4/CodeIgniter4/tests/system/Router/RouterTest.php:215
```
https://github.com/codeigniter4/CodeIgniter4/actions/runs/6457280498/job/17528438854?pr=7925